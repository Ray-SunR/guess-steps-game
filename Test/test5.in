game 50 60 test5A1.txt test5B1.txt
game 50 60 test5A1.txt test5B1.txt
game 50 60 test5A1.txt test5B1.txt
game 50 60 test5A1.txt test5B1.txt
game 50 60 test5A1.txt test5B1.txt
game 50 60 stdin stdin
50
game 50 60 test5A1.txt stdin
game 50 60 stdin test5B1.txt
game 320000 300000 stdin test5B2.txt
1
10
100
1000
10000
100000
game 320000 300000 test5A2.txt stdin
2
20
200
2000
20000
200000
game 320000 300000 stdin stdin
1
2
10
20
100
200
1000
2000
10000
20000
100000
200000
quit
