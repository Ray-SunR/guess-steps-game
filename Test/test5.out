Total is 50, max deduction is 60
Player A's move
Total is -10
Player A wins
Score is
A 1
B 0
Total is 50, max deduction is 60
Player B's move
Total is 0
Player B wins
Score is
A 1
B 1
Total is 50, max deduction is 60
Player A's move
Total is -10
Player A wins
Score is
A 2
B 1
Total is 50, max deduction is 60
Player B's move
Total is 0
Player B wins
Score is
A 2
B 2
Total is 50, max deduction is 60
Player A's move
Total is -10
Player A wins
Score is
A 3
B 2
Total is 50, max deduction is 60
Player B's move
Total is 0
Player B wins
Score is
A 3
B 3
Total is 50, max deduction is 60
Player A's move
Total is -10
Player A wins
Score is
A 4
B 3
Total is 50, max deduction is 60
Player B's move
Total is 0
Player B wins
Score is
A 4
B 4
Total is 320000, max deduction is 300000
Player A's move
Total is 319999
Player B's move
Total is 319997
Player A's move
Total is 319987
Player B's move
Total is 319967
Player A's move
Total is 319867
Player B's move
Total is 319667
Player A's move
Total is 318667
Player B's move
Total is 316667
Player A's move
Total is 306667
Player B's move
Total is 286667
Player A's move
Total is 186667
Player B's move
Total is -13333
Player B wins
Score is
A 4
B 5
Total is 320000, max deduction is 300000
Player B's move
Total is 319998
Player A's move
Total is 319997
Player B's move
Total is 319977
Player A's move
Total is 319967
Player B's move
Total is 319767
Player A's move
Total is 319667
Player B's move
Total is 317667
Player A's move
Total is 316667
Player B's move
Total is 296667
Player A's move
Total is 286667
Player B's move
Total is 86667
Player A's move
Total is -13333
Player A wins
Score is
A 5
B 5
Total is 320000, max deduction is 300000
Player A's move
Total is 319999
Player B's move
Total is 319997
Player A's move
Total is 319987
Player B's move
Total is 319967
Player A's move
Total is 319867
Player B's move
Total is 319667
Player A's move
Total is 318667
Player B's move
Total is 316667
Player A's move
Total is 306667
Player B's move
Total is 286667
Player A's move
Total is 186667
Player B's move
Total is -13333
Player B wins
Score is
A 5
B 6
